build-python.sh

```diff
@@ -38,6 +38,8 @@
 #                                   Compile 32-bit version.
 #     --cuda
 #                                   Compile CUDA version.
+#     --rocm
+#                                   Compile CUDA version.
```

Comment:
Very very minor suggestion, but this list is alphabetized... can you please move this (and the argument-handling below) to right before `--time-costs`, to preserve that ordering?

Sorry for not noticing it in earlier reviews.